modules/reviewing-cluster-status-from-the-openshift-cluster-manager.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// *installing/validating-an-installation.adoc
+
+[id="reviewing-cluster-status-from-the-openshift-cluster-manager_{context}"]
+= Reviewing the cluster status from the {cloud-redhat-com}
+
+You can review detailed information about the status of your cluster in the {cloud-redhat-com}.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. In the *Administrator* perspective, navigate to *Home* -> *Overview* -> *Details* -> *OpenShift Cluster Manager* to open the *Overview* page for the cluster in the {cloud-redhat-com}.
++
+[NOTE]
+====
+Alternatively, you can navigate to the link:https://cloud.redhat.com/openshift/[{cloud-redhat-com}] directly and select your cluster ID from the list of available clusters.
+====
+
+. In the *Overview* page, review the following information about your cluster:
++
+* vCPU and memory availability and resource usage
++
+* The cluster ID, status, type, location, and the provider name
++
+* Node counts by node type
++
+* Cluster version details, the creation date of the cluster, and the name of the cluster owner
++
+* The life cycle support status of the cluster
++
+* Subscription information including the service level agreement (SLA) status, the subscription unit type, the production status of the cluster, the subscription obligation and the service level
++
+* A cluster history
+
+. Navigate to the *Monitoring* page to review the following information:
+* A list of any issues that have been detected
++
+* A list of alerts that are firing
++
+* The Cluster Operator status and version
++
+* Cluster resource usage
+
+. Navigate to the *Insights* page to review the following information that is provided by Red Hat Insights:
```

Comment:
I'd drop "that is" here. I know we often add "that" for clarification, but here it seems clunky to me.

Reply:
Thank you for highlighting this. Resolved.